x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/ecs_mapping.json

```diff
@@ -2457,6 +2457,144 @@
             "ignore_above": 1024,
             "type": "keyword"
           },
+          "indicator": {
```

Comment:
These need to be updated with the latest from https://github.com/elastic/ecs/pull/1127 before this gets merged/released.

Reply:
Update: as of today, these are up to date with the RFC.